packages/mdc-drawer/_mixins.scss

```diff
@@ -149,13 +149,22 @@
 }
 
 @mixin mdc-drawer-fill-color($color) {
-  background-color: $color;
+  &.mdc-drawer--permanent,
+  .mdc-drawer__drawer {
+    @include mdc-theme-prop(background-color, $color);
+  }
 }
 
 @mixin mdc-drawer-ink-color($color) {
-  color: $color;
+  @include mdc-theme-prop(color, $color);
+
+  .mdc-list-item {
```

Comment:
Let's remove this rule block. There's already a separate set of `mdc-list-item-*` mixins, so our clients (and our demos) should just use those.

(We generally try to keep each component as isolated as possible to make them easier to maintain.)